src/ceph-disk/ceph_disk/main.py

```diff
@@ -4743,6 +4758,12 @@ def make_activate_parser(subparsers):
         action='store_true', default=False,
         help='activate the deactived OSD',
     )
+    activate_parser.add_argument(
+        '--cluster',
+        metavar='NAME',
+        default='ceph',
+        help='ceph cluster name (default: ceph)',
+    )
```

Comment:
Activate does not have a --cluster argument because it can be determined with the find_cluster_by_uuid() function